pkg/metrics/metrics.go

```diff
@@ -44,6 +44,13 @@ var (
 		Buckets:   []float64{1, 5, 10, 15, 30, 60, 120, 300, 600, 1800, 3600},
 	}, []string{"operation"})
 
+	ExternalAPICalls = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: promNSUpjet,
+		Subsystem: promSysResource,
+		Name:      "external_api_calls",
+		Help:      "The number of external API calls.",
+	}, []string{"service", "service_operation"})
```

Comment:
nit: Are `service` and `service_operation` nomenclature specific to AWS? Could you please check the counterparts in Azure & GCP? These labels will potentially be used by the other providers and the metric name and the label names become part of our API. Would something like `api_group` & `api` a more common name across the cloud service providers? What did they call these labels in the community provider?